components/aero_workflow/sources/new-company/new-company.mjs

```diff
@@ -0,0 +1,35 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  key: "aero_workflow-new-company",
```

Comment:
Can you please refer to this docs on [component key patterns](https://pipedream.com/docs/components/guidelines/#component-key-pattern)?